# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1364574072 14400
#      Fri Mar 29 12:21:12 2013 -0400
# Node ID a695ee2dc17e44fa7fae829babbc0b843d0aa090
# Parent  1834b91292ab25cf08ee1a2b512a9c83f2d8affa
* main-window.cc: Include <QDesktopWidget>

diff --git a/libgui/src/main-window.cc b/libgui/src/main-window.cc
--- a/libgui/src/main-window.cc
+++ b/libgui/src/main-window.cc
@@ -28,16 +28,17 @@ along with Octave; see the file COPYING.
 #include <QLabel>
 #include <QMenuBar>
 #include <QMenu>
 #include <QAction>
 #include <QSettings>
 #include <QStyle>
 #include <QToolBar>
 #include <QDesktopServices>
+#include <QDesktopWidget>
 #include <QFileDialog>
 #include <QMessageBox>
 #include <QIcon>
 
 #ifdef HAVE_QSCINTILLA
 #include "file-editor.h"
 #endif
 #include "main-window.h"
